src/main/scala/ch/epfl/bluebrain/nexus/iam/marshallers/instances.scala

```diff
@@ -34,22 +35,27 @@ object instances extends FailFastCirceSupport {
     Encoder.encodeString.contramap(fd => s"${fd.toMillis} ms")
 
   implicit val iamErrorEncoder: Encoder[IamError] = {
-    implicit val config = rejectionConfig
+    implicit val config: Configuration = rejectionConfig
```

Comment:
I think you can make the config implicit in the object scope, it's only use in these methods anyway.

Reply:
good point, will fix